### PR TITLE
infrastructure: add digital-rewards section

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

**Comment by RazvanN7** [comment]
cc @Jokeswar please review this and potentially add more information regarding the infrastructure and the guides to use the scripts you developed.

cc @gstefans as I have added some steps on how to obtain smileys, but those are not quite accurate.

File: chapters/infrastructure/digital-rewards/reading/README.md [hunk 1]
```diff
@@ -3,10 +3,38 @@
 Owner: Răzvan Nițu
 Reviewer: Mihai Băruța
```
Comment (teodutu) [inline chapters/infrastructure/digital-rewards/reading/README.md:4]:
```suggestion
```

File: chapters/infrastructure/digital-rewards/reading/README.md [hunk 2]
```diff
@@ -3,10 +3,38 @@
...
 
-definitions, types
+Some teaching paradigms involve the use of a reward system to incentivize learners to study.
+To enable the use of such teaching techniques, we offer an infrastructure for the use of cryptocurrency rewards that may be used to stimulate both learning and contributions to the educational content.
```
Comment (teodutu) [inline chapters/infrastructure/digital-rewards/reading/README.md:7]:
```suggestion
To enable such teaching techniques, we offer an infrastructure for cryptocurrency rewards with the purpose of stimulating learning and contributions to the educational content.
```

File: chapters/infrastructure/digital-rewards/reading/README.md [hunk 3]
```diff
@@ -3,10 +3,38 @@
...
+We do not discuss the efectiveness of reward-based learning techniques, we simply provide the tools to implement such a system if an educator chooses to use it.
```
Comment (teodutu) [inline chapters/infrastructure/digital-rewards/reading/README.md:8]: But shouldn't we discuss it? Why would someone use cryptocurrency if we don't make a case for it?

File: chapters/infrastructure/digital-rewards/reading/README.md [hunk 4]
```diff
@@ -3,10 +3,38 @@
...
 
-goals of using digital rewards
+Rewards may be used at different steps in the learning process:
 
-infrastructure for digital rewards
+- when a learner correctly completes a designated teaching activity: a drill or set of drills, an assignment, an exam etc.
+- when a learner contributes content to the existing repo
+- when a learner helps his peers (by publicly answering forum questions or helping them during practical sessions)
```
Comment (teodutu) [inline chapters/infrastructure/digital-rewards/reading/README.md:14]: Or when the learner reviews a contribution

File: chapters/infrastructure/digital-rewards/reading/README.md [hunk 5]
```diff
@@ -3,10 +3,38 @@
...
+- the awarding entity sets up [a wallet](https://wallet.smileyco.in/?network=smileycoin) and deposits some smiley in there.
+- the potential receivers also create a wallet.
+- once an activity for which smiley coins are awarded is completed, the awarding entity sends an amount of smiley coins to the receiver.
+
+For the awarding entity to be able to send the necessary smiley coins, it is required that the receivers wallet address is known.
+As a consequence, the receiver needs to publicize his wallet address.
+As such, for smileys that are awarded automatically, the receiver needs to have an account that is associated with his wallet on the platform where the activity is performed (either a github account for contributions or a platform account on the hosting service - such as TutorWeb).
+For smiley that are awarded manually, the receiver may simply communicate the receiving wallet address.
+
+Currently, we have implemented some scripts that make it easy to manually send smileycoin rewards.
```
Comment (teodutu) [inline chapters/infrastructure/digital-rewards/reading/README.md:40]: You should add links to them here [1], and they ought to be detailed in the Infrastructure chapter.

[1] https://github.com/Jokeswar/smileycoin-api/wiki/Teaching-Assistan-Handbook

**Review by teodutu — CHANGES_REQUESTED** [review]

File: chapters/infrastructure/digital-rewards/reading/README.md [hunk 6]
```diff
@@ -3,10 +3,38 @@
...
+- when a learner helps his peers (by publicly answering forum questions or helping them during practical sessions)
 
-cryptocurrency
+Choosing the items for which rewards are offered is a subjective decision depending on personal preference and cultural background.
+However, from a usability perspective, there are two categories of rewards:
+
+- those that can be awarded automatically (e.g. a learner solves a multiple choice question)
+- those that are awarded manually (e.g. a learner has answered a verbal question during a lecture)
+
+To account for these situations, we have developed an infrastructure that uses an educational cryptocurrency that may be awarded automatically or manually.
```
Comment (Jokeswar) [inline chapters/infrastructure/digital-rewards/reading/README.md:22]: Should we add some links to the infrastructure repositories?

- https://github.com/Jokeswar/smileycoin-api
- https://github.com/Jokeswar/smileycoin-node

**Review by Jokeswar — COMMENTED** [review]
I am unsure if we need to detail the inner workings of the smiley coin reward system that we have. I am referring to some diagrams or extra details on the technical side. @RazvanN7 what do you think about that?

File: chapters/infrastructure/digital-rewards/reading/README.md [hunk 7]
```diff
@@ -1,12 +1,51 @@
...
+To use the system, the following steps need to be taken:
+
+- The awarding entity sets up [a wallet](https://wallet.smileyco.in/?network=smileycoin) and deposits some smiley in there.
+- The potential receivers also create a wallet.
+- Once an activity for which smiley coins are awarded is completed, the awarding entity sends an amount of smiley coins to the receiver.
+
+We recommend the use of the [`Coinomi` wallet](https://github.com/open-education-hub/smileycoin-api/wiki/Student-Handbook#1-download--set-up-coinomi).
+It's popular and it can be used to store different cryptocurrencies, not just SmileyCoins.
+
+For the awarding entity to be able to send the necessary smiley coins, it is required that the receivers wallet address is known.
```
Comment (teodutu) [inline chapters/infrastructure/digital-rewards/reading/README.md:42]:
```suggestion
For the awarding entity to be able to send the necessary smiley coins, it is required that the receivers wallet address be known.
```
although a relic, the subjunctive is still used

**Review by teodutu — CHANGES_REQUESTED** [review]
You're using SmileyCoins, smiley coins, Smiley Coins and smiley at the same time which makes the text look unproffessional. Be consistent.

**Review by teodutu — APPROVED** [review]
Add "incentivize" and "cryptocurrenciee" to spellcheck's worldlist to silence thia workflow

**Comment by razvand** [comment]
@teodutu , do one final check, please.

**Review by teodutu — APPROVED** [review]
Right, I had forgotten about `config.yaml`.

**Pull request merged**